lib/Runtime/Library/amd64/JavascriptFunctionA.S

```diff
@@ -0,0 +1,144 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+.intel_syntax noprefix
+
+.extern _ZN2Js18JavascriptFunction13DeferredParseEPPNS_14ScriptFunctionE
+
+.global amd64_CallFunction
+.global	_ZN2Js18JavascriptFunction20DeferredParsingThunkEPNS_16RecyclableObjectENS_8CallInfoEz
```

Comment:
I also couldn't find a reference for this one? (Apart from the one defined under this source file)